UserManagement/views.py

```diff
@@ -109,13 +109,13 @@ def getAllDashboards(request):
         serializer = DashboardSerializer(dashboard, many = True)
         return Response(serializer.data)
 
-@api_view(['GET'])
+@api_view(['POST'])

```

Comment:
Why getDashboard is POST API?

Reply:
I think that the getDashboard api needs to be changed because it requires the user_id in order to fetch a certain dashboard of a user

Reply:
So, are we changing it? Change it to GET. You can pass parameters to GET API right?

Reply:
I don't think you can pass parameters to get API. Everytime I tried it let to errors. But I may be wrong.

Reply:
https://github.com/coskunlabGT/coskunLabDjangoApp/blob/master/quickOrder/message_views.py#L77

You pass request in the GET API. request has parameters.